functionalTest/test/utilities/reusableLibrary.js

```diff
@@ -1,6 +1,75 @@
-/**
- * Created this file to include all reusable methodes in future.
- */
-class reusableLibrary {}
+import dashboard from "../pageobjects/dashboardPage";
+import moment from "moment";
 
+class reusableLibrary {
+  sliderTrackResult() {
+    var fromDate = dashboard.fromDate.getText();
+    var toDate = dashboard.toDate.getText();
+
+    const TIMESPAN_ALL = "select-all";
+    const TIMESPAN_THREE_MONTHS = "select-last-three-months";
+    const TIMESPAN_ONE_MONTH = "select-last-month";
+    const TIMESPAN_TWO_WEEKS = "select-last-two-weeks";
+    const TIMESPAN_ONE_WEEK = "select-last-week";
+
+    function calculateLeftButtonPercentage(fromDate, toDate, timespan) {
+      let totalDays = moment(toDate).diff(moment(fromDate), "days") + 1;
+      let expectedDays;
+      switch (timespan) {
+        case TIMESPAN_ALL:
+          expectedDays = totalDays;
+          break;
+        case TIMESPAN_THREE_MONTHS:
+          expectedDays = 365 / 4;
+          break;
+        case TIMESPAN_ONE_MONTH:
+          expectedDays = 365 / 12;
+          break;
+        case TIMESPAN_TWO_WEEKS:
+          expectedDays = 14;
+          break;
+        case TIMESPAN_ONE_WEEK:
+          expectedDays = 7;
+          break;
+        default:
+          throw new Error("timespan invalid: " + timespan);
+      }
+      const timePeriodPercentage = (100 * expectedDays) / totalDays;
+      let leftPercent = 100 - timePeriodPercentage;
+      return leftPercent;
+    }
+
+    function valueCheck(expectedValue) {
+      var Style = dashboard.sliderTrack.getAttribute("style");
+      var actualValue = retrieveLeftWidth(Style, "left");
+      let diff = Math.abs(expectedValue - actualValue);
+      if (diff < 1) {
```

Comment:
This could be replaced with:
expect(diff).toBeLessThan(1);

Reply:
Replaced with suggested changes.